src/sandstorm/run-bundle.c++

```diff
@@ -652,6 +652,17 @@ public:
                   .build();
             },
             "For internal use only.")
+        .addSubCommand("admin-token",
+            [this]() {
+              return kj::MainBuilder(context, VERSION,
+                      "Generates a new admin token that you can use to access the admin settings "
+                      "page. This is meant for initial setup, or if an admin account is locked out.")
+                  .addOption({"short"}, [this]() { shortOutput = true; return true; },
+                      "Pass this flag if you wish to output only the token.")
```

Comment:
style nitpick: "Pass this flag if you wish to" seems redundant. Just "Output only the token." seems sufficient.